### ISSUE TITLE
tag or caption images

### ISSUE THREAD

**Issue by wwnbbd** [chuhaojin/bottom-up-attention-ForApp#4]
Thanks for your job!
I want to know how to use this model to tag or caption images, I find this code base only provide a model to extract visual and text feature. Do I need to prepare a tag list ?

**Comment by chuhaojin** [comment]
Yeah, for the image tagging task, you need to prepare a tag list.

**Comment by wwnbbd** [comment]
Will the tag list used in the paper be released ?
